Ryujinx/Ui/MainWindow.cs

```diff
@@ -44,6 +44,8 @@ public class MainWindow : Window
 
 #pragma warning disable CS0169, CS0649, IDE0044
 
+        [GUI] public MenuItem ExitMenuItem;
+        [GUI] public MenuItem UpdateMenuItem;
         [GUI] MenuBar        _menuBar;
```

Comment:
```suggestion
        [GUI] MenuBar         _menuBar;
```
and same for following lines